docs/src/modules/contribute/pages/getting-started.adoc

```diff
@@ -1,21 +1,20 @@
 = Getting started
 
-The Cloudstate Build uses `sbt` as its build tool.
+This page describes how to setup a development environment for contributing to and developing Cloudstate itself. If you are after documentation on how to develop Cloudstate services, see xref:../develop/index.adoc[Developing Cloudstate services].
```

Comment:
xrefs should use this syntax: `xref:module:filename.adoc`. This is currently failing the validation.

```suggestion
This page describes how to setup a development environment for contributing to and developing Cloudstate itself. If you are after documentation on how to develop Cloudstate services, see xref:develop:index.adoc[Developing Cloudstate services].
```